ros2topic/test/test_cli.py

```diff
@@ -372,6 +372,24 @@ def test_topic_echo(self):
             ), timeout=10)
         assert topic_command.wait_for_shutdown(timeout=10)
 
+    @launch_testing.markers.retry_on_failure(times=5, delay=1)
+    def test_topic_echo_once(self):
+        with self.launch_topic_command(
+        arguments=[
+            'echo', '--once',
+            '/chatter',
+            'std_msgs/msg/String'
+        ]
+        ) as topic_command:
+            assert topic_command.wait_for_output(functools.partial(
+                launch_testing.tools.expect_output, expected_lines=[
+                    re.compile(r"data: 'Hello World: \d+'"),
+                    '---'
+                ], strict=True
+            ), timeout=10)
```

Comment:
@craigh92 FYI: by making the output expectation strict, you've ensure that these lines and only these lines can be present for the test to pass.

Reply:
Thanks. This info will help with writing future tests :) As for this test, this is the correct behaviour.